newIDE/app/src/Profile/Subscription/SubscriptionDialog.js

```diff
@@ -687,7 +726,10 @@ export default function SubscriptionDialog({
                 i18n,
                 displayedSubscriptionPlanWithPricingSystems
               ).map(planSpecificRequirements => (
-                <AlertMessage kind="info">
+                <AlertMessage
+                  kind="info"
+                  key={planSpecificRequirements.substring(0, 25)}
```

Comment:
(could just use the index of the map here)

Reply:
map indices should never be used for the keys 😇